test/dynamo/test_utils.py

```diff
@@ -198,7 +200,7 @@ def test_dynamo_timed(self, mock_time, mock_time_ns):
                    restart_reasons=set(),
                    dynamo_time_before_restart_s=0.0,
                    has_guarded_code=True,
-                   remote_cache_time_saved_s=0,
+                   remote_cache_time_saved_s=None,
```

Comment:
I believe these changes from 0 to None are fine. We're not hitting the remote cache path on the OSS tests so the field is never populated. It's just that the previous code defaulted to 0 when not populated: https://github.com/pytorch/pytorch/blob/e6c5a7748516a2c2f3cffcfb4572603a0578280a/torch/_dynamo/convert_frame.py#L1073